.markdownlint.yaml

```diff
@@ -8,7 +8,4 @@ MD033: false
 MD041: false
 
 # MD013/line-length
-MD013:
-  line_length: 100
-  code_blocks: false
-  ignore_code_blocks: true
+MD013: false
```

Comment:
I think this is more trouble than it's worth... And for whatever reason, when truncated the nested ordered list below to fit line length, `mdformat` was then requiring that I indent it to three spaces? Which MkDocs doesn't render as a nested list?